modules/navier_stokes/doc/content/source/fvbcs/PINSFVFunctorBC.md

```diff
@@ -0,0 +1,32 @@
+# PINSFVFunctorBC
+
+!syntax description /FVBCs/PINSFVFunctorBC
+
+## Overview
+
+This object is very similar to [PCNSFVStrongBC.md]. However,
+this object is designed for use in tandem with an
+incompressible/weakly-compressible set of objects (see
+[PINSFVMomentumAdvection.md]) while [PCNSFVStrongBC.md] is meant for use in
+tandem with a fully compressible set of objects (see [PCNSFVKT.md]).
+
+A *seeming* difference between this object and [PCNSFVStrongBC.md] is that variable
```

Comment:
Do you mean you need to read more because you've never seen the word "seeming" before or because you're intrigued when someone writes the word "seeming" and the documentation is like a nail-biting, page-turning thriller?

Reply:
Your PRs are more like a never-ending scroll lol